tests/cases/helpers/case_helper_care_platform_admin_on_call.yaml

```diff
@@ -62,6 +62,38 @@ AdminOnCallRefreshDevice:
     - Type: sleep
       Time: 1
 
+AdminOnCallRefreshParticipant:
+  Vars:
+    LOGOUT_OPTION: "Go Off Shift"
+  Aftercases:
+    - CarePlatformLogout
+  Roles:
+    - Role: desktopChrome
+      App: desktop
+  Actions:
+    # Login into Care Platform
+    - Type: case
+      Value: AdminOnCallLogin
+    # Go to Participant Records
+    - Type: case
+      Value: CarePartnerParticipantRecordsNavigation
+    # Check for "Refresh" button
+    - Type: wait_for
+      Strategy: xpath
+      Id: $PAGE.care_platform_call_portal.refresh_button$
+    - Type: click
+      Strategy: xpath
+      Id: $PAGE.care_platform_call_portal.refresh_button$
+    # Refresh confirmation
+    - Type: wait_for
+      Strategy: xpath
+      Id: $PAGE.care_platform_call_portal.yes_refresh_dialog_button$
+    - Type: click
+      Strategy: xpath
+      Id: $PAGE.care_platform_call_portal.yes_refresh_dialog_button$
+    - Type: sleep
```

Comment:
Is this sleep necessary at the end?

Reply:
It was not need, I replaced the sleep for a wait on the following commit: https://github.com/neveralone-chs-org/TestRay/pull/227/commits/3722e53ca5805cc6490f2e733839af41942e2f7d